bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/store/mem/impl/MemHttpCacheStoreImpl.java

```diff
@@ -201,12 +232,35 @@ public long getTtl() {
     }
 
     @Override
+    public String getCacheEntry(String cacheKeyStr) throws IOException {
+        CacheKey cacheKey = null;
+
+        for (CacheKey cacheKeyTmp : cache.asMap().keySet()) {
```

Comment:
Couldnt find a clean way to collect the write params to create a synthetic CacheKey obj. Figured this should work 98% of the time .. ?